doc/guide/feature-virtualmachines.xml

```diff
@@ -0,0 +1,50 @@
+<?xml version="1.0"?>
+<!DOCTYPE chapter PUBLIC "-//OASIS//DTD DocBook XML V4.3//EN"
+        "http://www.oasis-open.org/docbook/xml/4.3/docbookx.dtd">
+<chapter id="feature-virtualmachines">
+    <title>Virtual Machines</title>
+
+    <para>Cockpit can manage virtual machines running on the host.
+        These can be accessed from menu via <emphasis>Virtual Machines</emphasis></para>
+
+    <para>Primary datasource is QEMU/Libvirt, access to Libvirtd is wrapped by the <emphasis>virsh</emphasis> tool.</para>
```

Comment:
Can you put in &lt;ulink&gt; tags around initial uses of QEMU and libvirt so people know what they are?

Reply:
Done